src/frontend/app/features/applications/routes/add-routes/add-routes.component.ts

```diff
@@ -33,7 +33,8 @@ import { Domain } from '../../../../store/types/domain.types';
 import { Route, RouteMode } from '../../../../store/types/route.types';
 import { ApplicationService } from '../../application.service';
 
-
+const hostPattern = '(([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9\-]*[a-zA-Z0-9])\.)*([A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9\-]*[A-Za-z0-9])';
+const pathPattern = `^((\/)${hostPattern}?(\/)*)*$`;
```

Comment:
I think the path allows more chars than the host. See: https://tools.ietf.org/html/rfc3986#section-3.3 and https://tools.ietf.org/html/rfc3986#appendix-A